apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-logo/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import './edit.scss';
+import edit from './edit';
+
+registerBlockType( 'a8c/site-logo', {
```

Comment:
I think you need an `attributes` key here for `addSiteLink` to work.

Reply:
It's registered in `class-full-site-editing.php`. I never intend to serialize this / need to parse this, so it should be okay for now. 

I was a little afraid of the weirdness of logo logic so I decided not to duplicate the output on the client.